### ISSUE TITLE
Content guide naming issue

### ISSUE THREAD

**Issue by coreycaitlin** [18F/federalist-docs#62]
Hello! As @line47 [noted over in the 18F Content Guide repo](https://github.com/18F/content-guide/issues/167), there is some confusion about the Federalist Content Guide vs. the 18F Content Guide. After some discussion with the content guild, we realized that the "Content guide" within the Federalist docs is actually more about how to use specific templates, rather than general content best practices.

To avoid confusion, we'd like to recommend changing the name of the "Content guide" within the Federalist docs to something else — perhaps "Template guide," "Using the templates," or "Page sections"?

**Comment by wslack** [comment]
Hey @coreycaitlin - I inherited the Federalist Content Guide, which was [originally written by Mel and Andre in summer '15](https://github.com/18F/federalist-content-guide/commits/18f-pages). I don't have a strong opinion on it and have not included it in user research.

In some ways, I agree with @line47 that having multiple content guides is redundant, and I think some of the Federalist content is outdated (no longer applying to the templates we have) and could be updated with USWDS guidance or something.

Thoughts?

**Comment by line47** [comment]
As part of the work on the docs site I'd like to we point to the 18F content guide for content recommendations and remove the content related pages on the federalist docs site.

Most of this section is really outdated and I'm not even sure if is is helpful.
I'm voting for removing the content section altogether

**Comment by wslack** [comment]
@line47 I'll defer to @OpenGlobe since he's doing content work for us. I wouldn't want to do a wholesale removal unless we're sure there's nothing useful there - and if memory serves, the site still might be useful for someone creating a new site on Federalist.

**Comment by line47** [comment]
I'm going to break down the contents of the content guide pages here: 

### Content guide
This is a general overview about what to expect on the following pages.

### Writing content for your homepage
This page seems to be geared towards a particular template as it talks about very specific elements and sections. I'm wondering if this type of guidance would be better served on a wiki within a repo?

### Writing about your team
Talks about writing an about page or team page.
I'm wondering if this type of guidance would be better served on a wiki within a repo?

### Writing about your research
I'd love some thoughts on this page and how it came about. 
I'm wondering if this type of guidance would be better served on a wiki within a repo?

### Writing clear contact forms
I'm not sure if this is referring to a particular template or what? 

### Best practices for content
Basically a link to the 18F content guide. I think this could be worked into the [customer responsibilities](https://federalist-docs.18f.gov/pages/using-federalist/customer-responsibilities/) page under the You own your content section. 👍 

### Adding search to your site
Not much info here. Explains that you can go to search.gov to get a search account 
I think this can be described in the repo along with other front end configurations

CC @OpenGlobe

**Comment by wslack** [comment]
The wiki'able stuff.....I wouldn't want any TTS content intended for the public to be on a GitHub wiki. We can also ask the content guild before a kibosh.....maybe I'm just nervous because I don't feel like Federalist "owns" those pages though it does.

**Comment by OpenGlobe** [comment]
I generally like the idea of moving some of that material to a wiki @line47.  I also think that we might be able to rely more heavily on the 18F Content Guide in order to keep things more consistent.

**Comment by wslack** [comment]
I'm fine with relying on the18F content guide and deleting unimportant content, but admit that I still struggle with putting anything in a wiki.

**Comment by line47** [comment]
So can we then just keep the single content page that points over to the 18F content guide?

**Comment by wslack** [comment]
Sure; I think so!

**Comment by line47** [comment]
PR https://github.com/18F/federalist-docs/pull/183 addressed this issue
